src/rgw/rgw_civetweb.cc

```diff
@@ -120,6 +120,10 @@ void RGWCivetWeb::init_env(CephContext *cct)
     env.set("REMOTE_USER", info->remote_user);
   }
 
+  if (info->remote_addr) {
```

Comment:
Even if `remote_add` is empty, it's still fine(ops log entry default value is empty string), I'm fine with adding length check here, the concern is if we've introduced the length check, should we also do this for `info->remote_user`? 

So in order to keep consistency with surrounding code, I'd prefer we don't add length check.